config/routes.rb

```diff
@@ -9,4 +9,5 @@
   post "signup", to: "users#create"
   root "static_pages#home"
   resources :users
+  resources :account_activations, only: [:edit]
```

Comment:
ở đây dùng `%i()` nhé em, em đọc thêm về ruby percent notation nhé
```suggestion
  resources :account_activations, only: %i(edit)
```